airflow/opt/providers/etna/etna/plugins/impersonate_user_view.py

```diff
@@ -0,0 +1,54 @@
+from flask import flash
+
+from wtforms import SelectField
+from wtforms.validators import InputRequired
+from flask_appbuilder.fieldwidgets import Select2Widget
+from flask_appbuilder.forms import DynamicForm
+
+
+class ImpersonateUserForm(DynamicForm):
+    email = SelectField(label='User',
+        validate_choice=False, # Seems to be required to work with Airflow, since the choices are injected
+        description=('User whose permissions to impersonate'),
+        validators = [InputRequired()],
+        widget=Select2Widget())
+
+
+try:
+    from flask_appbuilder import SimpleFormView
+    from flask_appbuilder.security.decorators import has_access
```

Comment:
Just to clarify: basically, access to this view requires whitelisting the view in, which only the admin role has "sudo" privilege to view even without explicitly adding it to that role, right?

Reply:
Yes, only the Admin role has permissions for this Impersonate view, and it is implied (do not have to add it explicitly).